site/src/components/button.md

```diff
@@ -92,11 +92,11 @@ Warning buttons are yellow on most themes. This color indicates a critical situa
 - The user input was validated and a minor problem occurred. The user can continue without fixing the problem, but might lead to an error later on.
 - A message contains information about a warning.
 
-| Type     | Class                                           | Default State                                                        | Hover                                                                           |
-| -------- | ----------------------------------------------- | -------------------------------------------------------------------- | ------------------------------------------------------------------------------- |
-| Outlined | `.button`, `.button-warning`                    | <button class="button button-warning">Default</button>               | <button class="button button-warning is-hovered">Default</button>               |
-| Clear    | `.button`, `.button-warning`, `.button-clear`   | <button class="button button-warning button-clear">Default</button>  | <button class="button button-warning button-clear is-hovered">Default</button>  |
-| Filled   | `.button` , `.button-warning`, `.button-filled` | <button class="button button-warning button-filled">Default</button> | <button class="button button-warning button-filled is-hovered">Default</button> |
+| Type     | Class                                           | Default State                                                       | Hover                                                                          | Disabled                                                                     |
+| -------- | ----------------------------------------------- | ------------------------------------------------------------------- | ------------------------------------------------------------------------------ | ---------------------------------------------------------------------------- |
+| Outlined | `.button`, `.button-warning`                    | <button class="button button-warning">Default</button>              | <button class="button button-warning is-hovered">Default</button>              | <button class="button button-warning" disabled>Default</button>              |
+| Clear    | `.button`, `.button-warning`, `.button-clear`   | <button class="button button-warning button-clear">Clear</button>   | <button class="button button-warning button-clear is-hovered">Clear</button>   | <button class="button button-warning button-clear" disabled>Default</button> |
```

Comment:
Disabled Clear button should be "Clear" instead of "Default"

Reply:
👍 Updated